Cannot update reason-react to 0.10.1

I am trying to use this template in my project.

I installed a newer `reason-react` 0.10.1 which introduced `dune` files in its source, so it looks like it can be recognized as a library by `dune`. The template has a `dune` file that makes `node_modules/reason-react` into a library like:

```
(subdir
 node_modules
 (dirs reason-react)
 (vendored_dirs reason-react)
 (subdir
  reason-react/src
  (include_subdirs unqualified)
  (library
   (name reason_react)
   (wrapped false)
   (modes melange))))
```

With this `dune` file, I get an error:
```
$ make build 
opam exec -- dune build @react 
Error: Library reason_react is defined twice:
- dune:8
- node_modules/reason-react/src/dune:1
make: *** [Makefile:30: build] Error 1
```
which makes sense because both of these dune files declare `reason_react` library.

What i did is remove the content of project's `dune` file, but then I get:
```
make build 
opam exec -- dune build @react 
File "src/dune", line 4, characters 20-32:
4 |  (libraries melange reason_react)
                        ^^^^^^^^^^^^
Error: Library "reason_react" not found.
```

So the dune build both sees this library and does not see it.
I tried to put into dune to discover that node_modules/reason-react contains `reason_react` package:
```
(dirs :standard node_modules/reason-react)
```
but this does not help.

What am I missing? Do i need to put into `project.opam` some information to look for `node_modules/reason-react/src/dune` ? Or do I need to add `bsconfig.json` (this file is not in the melange-opam-tempalte)

--- a/dune
+++ b/dune
@@ -1,11 +1,12 @@
 (subdir
  node_modules
  (dirs reason-react)
- (vendored_dirs reason-react)
+ (data_only_dirs reason-react)
  (subdir
   reason-react/src
   (include_subdirs unqualified)
   (library
    (name reason_react)
    (wrapped false)
-   (modes melange))))
+   (modes melange)
+   (flags -w -9-27-32-20))))

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "dependencies": {
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "reason-react": "^0.10.0"
+    "reason-react": "^0.10.1"
   },
   "devDependencies": {
     "webpack": "^5.73.0",

